Platforms/OSX/AppKit/NSApplication.swift

```diff
@@ -710,7 +710,7 @@ enum NSApplicationDelegateReply : UInt {
 }
 extension NSApplication {
   var windowsMenu: NSMenu?
-  func arrangeInFront(sender: AnyObject?)
+  func arrangeIn(front sender: AnyObject?)
```

Comment:
Yeah, that's a pretty strong indicator that we shouldn't split. In these cases, "sender" would actually make a good argument label, because "arrangeInFront" implies nothing about what the argument is.